protos/format.proto

```diff
@@ -100,6 +100,18 @@ message Manifest {
   // 
   // For a single file, will be zero.
   uint32 max_fragment_id = 11;
+
```

Comment:
ooc, is there any reason to prefer having a transaction file over just putting the transaction directly in the manifest?

Reply:
The main reason is that outside of the commit loop, which only happens in a narrow time range, this information is mostly redundant. In theory this can be quite large, so I didn’t want us to have to pay the IO cost every time the version was loaded.

Reply:
I suppose since many times it's small, a classic compromise would be to allow it to be inlined if it isn't too large. Does that sounds better?

Reply:
I think either way is fine tbh. Write txs should be able to endure one additional IO. We can keep it as is for simplicity.

Reply:
I'm going to forgo this for now, since I think we can add this in the future.